validate.js

```diff
@@ -0,0 +1,31 @@
+const colors = require('colors/safe');
+const Validator = require('jsonschema').Validator;
+
+const filters = require('./config/filters.json');
+const canonical = require('./config/canonical.json');
+
+
+const filtersSchema = require('./schema/filters.json');
+const canonicalSchema = require('./schema/canonical.json');
+
+validateSchema('config/filters.json', filters, filtersSchema);
+validateSchema('config/canonical.json', canonical, canonicalSchema);
+
+// Perform JSON Schema validation
+function validateSchema(fileName, object, schema) {
+    const v = new Validator();
+    const validationErrors = v.validate(object, schema).errors;
+    if (validationErrors.length) {
+        console.error(colors.red('\nError - Schema validation:'));
+        console.error('  ' + colors.yellow(fileName + ': '));
+        validationErrors.forEach(e => {
+            if (e.property) {
+                console.error('  ' + colors.yellow(e.property + ' ' + e.message));
+            } else {
+                console.error('  ' + colors.yellow(e));
+            }
+        });
+        console.error();
+        process.exit(1);
+    }
+}
```

Comment:
Good idea, you should add a build test, and validate that the newly-build files are the same that were committed!

Reply:
I don't really get what you mean... When should this script run?

Reply:
Well, the build only functions if the json is valid anyway (i.e. see what `npm run build` does) - and if the contributor actually ran the build locally and committed the results, the build files (`/dist/`) shouldn't change after the build.

This travis config will already prevent an array of problems, though, so good job!